command/catalog/list/services/catalog_list_services_test.go

```diff
@@ -34,6 +36,7 @@ func TestCatalogListServicesCommand(t *testing.T) {
 	t.Parallel()
 	a := agent.NewTestAgent(t.Name(), ``)
 	defer a.Shutdown()
+	testrpc.WaitForLeader(t, a.RPC, "dc1")
```

Comment:
Thank you for this.

Can you please replace WaitForLeader here with WaitForTestAgent? WaitForLeader isn't resilient enough when I tested it locally.

Reply:
@freddygv DONE